ecosystem/typescript/sdk/src/providers/indexer.ts

```diff
@@ -0,0 +1,77 @@
+import axios from "axios";
+
+import { AnyNumber } from "../bcs/types";
+import { MaybeHexString } from "../hex_string";
+import { GetAccountCurrentTokensQuery, GetTokenActivitiesQuery } from "../indexer/generated/operations";
+import { GetAccountCurrentTokens, GetTokenActivities } from "../indexer/generated/queries";
```

Comment:
it uses only `graphql-request` and only when generating code, since we generate it before publishing, i think we should be good with having it in devDependencies